api/src/main/java/org/openmrs/module/fhir/api/impl/ConditionServiceImpl.java

```diff
@@ -18,16 +18,15 @@
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.hl7.fhir.dstu3.model.Condition;
-import org.openmrs.api.context.Context;
+import org.openmrs.api.impl.BaseOpenmrsService;
 import org.openmrs.module.fhir.api.ConditionService;
 import org.openmrs.module.fhir.api.db.FHIRDAO;
 import org.openmrs.module.fhir.api.strategies.condition.ConditionStrategyUtil;
-import org.openmrs.module.fhir.api.util.FHIRConditionUtil;
+import org.openmrs.module.fhir.api.util.ContextUtil;
 
-import java.util.ArrayList;
 import java.util.List;
 
-public class ConditionServiceImpl implements ConditionService {
+public class ConditionServiceImpl extends BaseOpenmrsService implements ConditionService {
```

Comment:
What was the reason for extending BaseOpenmrsService?

Reply:
Just confirmed I don't need at all